images/win/scripts/Installers/Install-Vsix.ps1

```diff
@@ -0,0 +1,20 @@
+###################################################################################
+##  File:  Install-Vsix.ps1
+##  Desc:  Install the Visual Studio Extensions from toolset.json
+###################################################################################
+
+$ErrorActionPreference = "Stop"
+
+$toolset = Get-ToolsetContent
+$vsixPackagesList = $toolset.visualStudio.vsix
+if(-not $vsixPackagesList) {
+    Write-Host "No extensions to install"
+    exit 0
+}
+
+$VsVersion = $toolset.visualStudio.Version
+$vsixPackagesList | ForEach-Object {
+    Install-VsixExtension -url $_.url -name $_.name -VsVersion $VsVersion

```

Comment:
nitpick: different case notation for parameter names

Reply:
case of 'v' has been fixed